### PR TITLE
Clears Old Storage for Session pallet

### PR DESCRIPTION
**Pull request by gupnik**
Fixes https://github.com/paritytech/polkadot/issues/4691

The old migration didn't clear the data properly. This PR adds a V2 to clear everything under Session::HistoricalSessions and Session::StoredRange

As discussed in https://github.com/paritytech/substrate/pull/13997, this migration is added directly to Polkadot and Kusama.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2724366

**Review by eskimor — APPROVED** [review]

**Review by ordian — COMMENTED** [review]
Missing Kusama?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2732103

**Comment by gupnik** [comment]
bot fmt

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2732305

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2732398

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2732486

File: runtime/common/src/session/migration.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	storage::storage_prefix,
+	traits::{Get, OnRuntimeUpgrade},
+	weights::Weight,
+};
+use pallet_session::Config;
+use sp_io::{storage::clear_prefix, KillStorageResult};
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+pub struct MigrateToV2<T>(sp_std::marker::PhantomData<T>);
```
Comment (bkchr) [inline runtime/common/src/session/migration.rs:27]: Name is misleading and should get some docs on what it is doing.

**Review by bkchr — APPROVED** [review]

**Comment by gupnik** [comment]
bot fmt

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2739511

**Comment by gupnik** [comment]
bot merge

**Comment by gupnik** [comment]
bot merge force

**Pull request merged**